gradle/libs.versions.toml

```diff
@@ -30,7 +31,9 @@ airbnb-showkase = { module = "com.airbnb.android:showkase", version.ref = "showk
 airbnb-showkase-processor = { module = "com.airbnb.android:showkase-processor", version.ref = "showkase" }
 
 android-documentation-plugin = { module = "org.jetbrains.dokka:android-documentation-plugin", version.ref = "androidDocumentationPlugin" }
+# Artifact: https://mvnrepository.com/artifact/androidx.appcompat/appcompat
```

Comment:
IMO this info is useless. We should either redirect to a public github project or changelog. Also, IntelliJ now directly shows the available updates.

Reply:
I personally use androidx.tech to get the changelog & the sources is it good enough or should we reference the androidx Github?
c995245